### PR TITLE
libcdr 0.1.2

### PR DESCRIPTION
**Pull request by luzpaz**

### CODE REVIEW

**Comment by luzpaz** [comment]
Added cppunit dep since building 0.1.2 required it
```configure: error: Package requirements (cppunit) were not met:
No package 'cppunit' found
```
You can also find cppunit used in another librevenge suite formula, libvisio. See https://github.com/Homebrew/homebrew/blob/master/Library/Formula/libvisio.rb#L16

File: Library/Formula/libcdr.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 class Libcdr < Formula
   desc "C++ library to parse the file format of CorelDRAW documents"
   homepage "https://wiki.documentfoundation.org/DLP/Libraries/libcdr"
-  url "http://dev-www.libreoffice.org/src/libcdr/libcdr-0.1.1.tar.bz2"
-  sha256 "72fe7bbbf2275242acdf67ad2f9b6c71ac9146a56c409def360dabcac5695b49"
+  url "http://dev-www.libreoffice.org/src/libcdr/libcdr-0.1.2.tar.bz2"
+  sha256 "d05a986dab9f960e64466072653a900d03f8257b084440d9d16599e16060581e"
   revision 2
```
Comment (DomT4) [inline Library/Formula/libcdr.rb]: Please remove revision lines outside any bottle block on stable version updates, Thanks!

File: Library/Formula/libcdr.rb [hunk 2]
```diff
@@ -14,10 +14,12 @@ class Libcdr < Formula
 
   depends_on "pkg-config" => :build
   depends_on "boost" => :build
+  depends_on "cppunit" => :build
   depends_on "icu4c"
   depends_on "librevenge"
   depends_on "little-cms2"
 
+
```
Comment (DomT4) [inline Library/Formula/libcdr.rb]: Don't need this extra line.

**Comment by luzpaz** [comment]
fixed

**Comment by DomT4** [comment]
Merged in https://github.com/Homebrew/homebrew/commit/8197f31a1413f62844ebbdb426c3806ea0e0419f. Thanks again!

**Pull request closed without merging**